docs/src/typings.d.ts

```diff
@@ -11,6 +11,7 @@ declare module '@retailmenot/anchor' {
   const AutoComplete: any;
   const Badge: any;
   const Button: any;
+  const Card: any;
```

Comment:
We actually ship anchor with its own typings file so this shouldn't be needed so long as the client is on the latest version of Anchor. Also, typing anything `any` isn't a great idea. It's good for accelerating against a tough deadline, but it's not a sustainable practice.

Reply:
I'll look into this. If we're shipping the types, it's really odd they aren't coming through. Agreed about `any`, this list has just been building and building. Need to kill the trend.

Reply:
For posterity, @sharkparty discovered that removing the Anchor type definitions in the repo solves the issue. Most likely these were given priority over the definitions in the Anchor module.